packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -36,12 +37,18 @@ BuySellModalFooter.propTypes = {
 
 const BuySellModal = observer(({ table_type, selected_ad, should_show_popup, setShouldShowPopup }) => {
     const isMounted = useIsMounted();
-    const { general_store, order_store } = useStores();
+    const { buy_sell_store, general_store, order_store } = useStores();
+    const { local_currency } = buy_sell_store?.advert || {};
```

Comment:
The issue could also be because we're using an empty object.

Reply:
Checked, it's not the reason